data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	"Monsters. Elves. Dragons. Squirrels." Kormak says.`
+			`	"Although we are not entirely indepedant, oh no. We are proud members of, and modestly funded by, the Society of Gentlemanly Exemplary Exploration, which was founded by the esteemed Dr. Henry Forsyth who cured the Rigellian Plague. SOGEE for short."`
+			`	"Soggy." Kormak pronouces.`
+			`	Tavish makes a pained expression as he says, "It is a most unfortunate pronunication for our esteemed society, but yes, SOGEE is what most call it. Hopefully this has satisfactorily answered your question?"`
+				goto questions2
+			
+			label india1
+			action
+				set "india1"
+			`	Tavish clears his throat, and then says quietly, "New India is a very recently colonized world. It is filled with life, and the remnants of life that have yet to be discovered."`
+			`	He leans in and says even more quietly, "On a recent expedition Kormak and I discovered what we believe to be remains of a crashed ship. A xeno-ship."`
+			`	"Aliens." Kormak adds helpfully.`
+			`	Tavish straightens up, "That is what we're after, and why we need your help! So you see you really are on the cutting edge of discovery with us. A fellow adventurer and explorer!"`
+			`	"Soggy." Kormak affirms.`
+				goto questions2
+			
+			label equipment1
+			action
+				set "equipment1"
+			`	"The deep jungles of New India are filled with aggressive tree-analogs." Tavish explains.`
+			`	"Monsters." Kormak says with bared teeth.`
+			`	"Unlike our more civilized trees on Earth, these ones are significantly more mobile. They will steal and smother anything left out, including pack animals and people."`
+			`	Kormak removes his cap and says solemnly, "Gerald."`
+			`	Tavish nods somberly, "Indeed, poor Gerald. If only his legs were as quick as his mind."`
+			`	"More cardio." Kormak agrees.`
+			`	"We had a special mechanical ring made to surround our campsite in ignited heat and prevent the tree-analog's tendrils from entering or interfering with our work."`
+			`	"Firewall." Kormak adds.`
+			`	"There was also a drill and sifter specially designed and programmed by Dr. Gerald to help uncover the vessel without damaging it," Tavish continues, "and, lastly, an underground sonar headset that would allow us to see its buried structure and avoid injuring it during our excavation."`
+			`	Tavish shrugs. "There were other things like tents, and picks, and shovels and what not. But all of that we can get on world. The ring, drill, and headset are all irreplaceable and absolutely necessary for this dig."`
+			`	"Irreplaceable." Kormak confirms.`
+				goto questions2
+
+			label end
+			action
+				clear "who1"
+				clear "india1"
+				clear "equipment1"
```

Comment:
Ah, I see how this works. These condition names might be okay, then, but I'd still make them more descriptive if possible.

Reply:
But why?